aladdin-container.sh

```diff
@@ -19,24 +19,6 @@ export PY_MAIN
 
 source "$SCRIPT_DIR/shared.sh"
 
-# Test user's aws configuration
```

Comment:
I went back and forth on this, but I think it ended up being cleaner with aladdin being more unopinionated on this stuff